controller/client/client.go

```diff
@@ -123,15 +122,14 @@ func (c *Client) UpdateRules(rulesSet *api.RulesSet) (interface{}, error) {
 // DeleteRules deletes the rules
 func (c *Client) DeleteRules(filter *api.RuleFilter) ([]byte, error) {
 
-	path := "/v1/rules"
+	path := rulesPath
```

Comment:
You should be able to condense this logic down to a single line:
`path := rulesPath + c.filterToStringQuery(filter)`